lib/sycamore/sycamore/data/element.py

```diff
@@ -174,9 +174,10 @@ def table(self) -> Optional[Table]:
         return self.data["table"]
 
     @table.setter
-    def table(self, value: Table) -> None:
+    def table(self, value: Optional[Table]) -> None:
         self.data["table"] = value
-        self.data["text_representation"] = self.data["table"].to_csv()
+        if value:
```

Comment:
might write out the `if value is not None` just in case we add a bool coercion to tables. Not sure such a thing ever makes sense so feel free to ignore me.